### DISCUSSION TITLE
Ignore island permission

### DISCUSSION THREAD

**Discussion by Cr4zyGeek** [BG-Software-LLC/SuperiorSkyblock2#1296] [Implemented Feature Requests] [answered]
Create a permission that automatically hide an island in the TOP if a member have this permission

[the remark this answers is not in the document]
**Reply by Cr4zyGeek** [reply]
Yes but it's not automatic if someone got a new rank with a particular permissions to do that

[the remark this answers is not in the document]
**Reply by OmerBenGera** [reply]
> Yes but it's not automatic if someone got a new rank with a particular permissions to do that

It will never be automatically, as the plugin will need to check everytime you open the top islands if any of the members has the permission - will hurt performance.

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
